test/unit/metrics/test_metrics_engine_dicts.py

```diff
@@ -109,3 +130,18 @@ def test_metric_group_summary_smoke(func):
 def test_derived_metrics_smoke(func):
     result = func(y_true, y_pred, sensitive_features=sf_binary)
     assert result == pytest.approx(derived_metric_results[func])
+
+
+@pytest.mark.parametrize("func", metric_group_summary_results_pos_label.keys())
+def test_metric_group_summary_pos_label_0(func):
+    # We're going to set pos_label=0, so for simplicity invert the previous inputs
+    y_true_invert = [1-y for y in y_true]
+    y_pred_invert = [1-y for y in y_pred]
+    result = func(y_true_invert, y_pred_invert, sensitive_features=sf_binary, pos_label=0)
+    assert result.overall == pytest.approx(
```

Comment:
you can just use `metric_group_summary_results[func]` everywhere below, right?

Reply:
I suppose I could, but I thought it would be more natural to use the same dictionary as provides the test data.